packages/dashboard/src/views/Holdings.js

```diff
@@ -100,17 +100,38 @@ const Holdings = ({
 
       const currAllData = historicQuotes['all']
       const currAllResults = currAllData.results[currAllData.results.length - 1]
-
-      // override current balance
+    
+      // Override cash value (buying power)
       smallStats[0] = {
         ...smallStats[0],
+        value: Math.round(selectedAccount.buying_power.amount * 100) / 100
+      }
+
+      // Override securities value
+
+      const positions = selectedAccount.positions
+    
+      let initialValue = 0
+      let securitesValue = positions.reduce(function (accumulator, currentValue) {
+          return accumulator + currentValue.todays_earnings_baseline_value.amount;
+      }, initialValue)
+
+      
+      smallStats[1] = {
+        ...smallStats[1],
+        value: Math.round(securitesValue * 100) / 100
+      }
+
+      // override current balance
+      smallStats[2] = {
+        ...smallStats[2],
         value: Math.round(currDayResults.value.amount * 100) / 100,
       }
 
       // override day gain
       const dayAmount = currDayResults.relative_equity_earnings.amount
-      smallStats[1] = {
-        ...smallStats[1],
+      smallStats[4] = {
```

Comment:
should this be moved to after `smallStats[3]` just to keep logical order

Reply:
yes, yes it should